owncloudApp/src/main/java/com/owncloud/android/extensions/ActivityExt.kt

```diff
@@ -334,6 +341,33 @@ private fun Activity.showSelectSecurityDialog(
     }
 }
 
+fun Activity.feedbackBehaviour(feedbackMail: String) {
```

Comment:
I don't like the name of this extension function... I think it doesn't express correctly what it is doing. What is "feedback behaviour"? It sounds like it is going to return a value among several ones in an `enum`, and that's not the case. Think about a new more suitable name

Reply:
I changed the name to: `sendEmailOrOpenFeedbackDialogAction`